src/UserModE131.h

```diff
@@ -0,0 +1,67 @@
+#include <ESPAsyncE131.h>
+
+
+
+class UserModE131:public Module {
+
+public:
+
+  UserModE131() :Module("e131/sACN support") {
+    print->print("%s %s\n", __PRETTY_FUNCTION__, name);
+
+    print->print("%s %s %s\n", __PRETTY_FUNCTION__, name, success?"success":"failed");
+  };
+
+  //setup filesystem
+  void setup() {
+    Module::setup();
+    print->print("%s %s\n", __PRETTY_FUNCTION__, name);
+
+    if(!net->interfacesInited) {
+      print->print("No network yet, can't start e131");
+    }
+
+//    print->print("%s %s %s\n", __PRETTY_FUNCTION__, name, success?"success":"failed");
+  }
+
+  void loop(){
+    // Module::loop();
+    if(!net->interfacesInited) {
```

Comment:
Feels like the usermods should be notified when we have a network connection. This actually fails when you make that field private

Reply:
Modules.h has a function called connected which calls all module->connected(), this is a virtual function in Module.h which can be implemented in each module, apparently if not implemented then it crashes (I hoped the virtual function is then called). 
As a workaround in SysModNetwork web->connected2(); is called (twice). If I replace by mdls->connected() the crash occurs, but maybe you can make it work by not calling unimplemented connected() functions?
But you can also use the workaround for the time being by adding e131mod->connected2() and implement connected2() in your module (dirty however on the long run)

Reply:
I now realize that it worked before the header file split up, that's kind of strange, why shouldn't it work now ?

Reply:
My understanding of virtual is that it's similar to abstract in java. You are effectively adding something to the interface, but the class needs to be extended to include the implementation. The way I used it (for the first time) in the Artimatix stuff means the you get a compile error if you don't implement rather than runtime.
If it's optional, then just being regular method that can get overriden should be fine

Reply:
Just looked at the docs, it's not quite the same, virtual allows for polymorphism rather than just enforcing other classes to implement

Reply:
Not sure what's going on. Gave up on fixing the connected issue, so tried calling the module directly, but that fails, even just calling `e131mod->name` from inside `handleConnection` fails